core/src/banking_stage/consume_worker.rs

```diff
@@ -107,7 +124,16 @@ impl ConsumeWorker {
 
     /// Send transactions back to scheduler as retryable.
     fn retry(&self, work: ConsumeWork) -> Result<(), ConsumeWorkerError> {
-        let retryable_indexes = (0..work.transactions.len()).collect();
+        let retryable_indexes: Vec<_> = (0..work.transactions.len()).collect();
+        let num_retryable = retryable_indexes.len();
+        self.metrics
+            .count_metrics
+            .retryable_transaction_count
+            .fetch_add(num_retryable, Ordering::Relaxed);
+        self.metrics
+            .count_metrics
+            .retryable_expired_bank_count
+            .fetch_add(num_retryable, Ordering::Relaxed);
```

Comment:
is `num_retryable` the right count to add to `retryable_expired_bank_count`?

Reply:
yeah, in this case (`ConsumeWorker::retry`), all transactions are retryable because the bank has expired or was not available, so these counts are equally updated **here**.
`retryable_transaction_count` is also updated on other retryable reasons that affect individual transactions in `update_on_execute_and_commit_transactions_output`

Reply:
Thanks for explaining.